examples/pxScene2d/src/pxScene2d.h

```diff
@@ -1451,7 +1451,7 @@ class pxScene2d: public rtObject, public pxIView
     return e;
   }
 
-  void sceneContainerDisposed(pxSceneContainerRef ref);
+  void innerObjDisposed(rtObjectRef ref);
```

Comment:
Please use full names as opposed to abbreviations when creating variables and methods.  Please change innerObjDisposed to innerObjectDisposed.